src/multiuser.cpp

```diff
@@ -147,14 +147,14 @@ class UserSentry {
         int buflen = sysconf(_SC_GETPW_R_SIZE_MAX);
         if (buflen < 0) {buflen = 16384;}
         std::vector<char> buf;
-        buf.reserve(buflen);
+        buf.resize(buflen);
```

Comment:
It's fine how you have it, but alternatively you could set the size in the constructor directly, eg:

```
        std::vector<char> buf(buflen);
```

instead of doing it in a separate resize.

(github is not letting me make a suggestion including the previous line, apparently because it's not part of the change set)

Reply:
Changed it. Thanks for the suggestion.